Cargo.toml

```
@@ -38,3 +38,4 @@ debug-assertions = true
 default = ["min-valkey-compatibility-version-8-0"]
 enable-system-alloc = ["valkey-module/enable-system-alloc"]
 min-valkey-compatibility-version-8-0 = []
+valkey_8_0 = []  # Empty feature flag for Valkey 8.0
```

You can add something like this to provide more context.

```suggestion
valkey_8_0 = []  # valkey-bloom is intended to be loaded on server versions >= Valkey 8.1 and by default it is built this way (unless this flag is provided). It is however compatible with Valkey version 8.0 if the user explicitly provides this feature flag in their cargo build command.
```